src/components/SearchArtworks.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from "react";
+import apiService from "../services/apiService";
+import "../styles/searchArtwork.css"; 
+
+import { Container, Row, Col, Card, Form, Button } from "react-bootstrap";
+
+const SearchArtworks = () => {
+  const [query, setQuery] = useState("");
+  const [artworks, setArtworks] = useState([]);
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState(null);
+
+  // Array of popular artists for search options
+  const artists = ["Rembrandt", "Vermeer", "Van Gogh", "Frans Hals", "Jan Steen", "Jacob van Ruisdael"];
+
+  // Function to handle search
+  const handleSearch = async (e) => {
+    e.preventDefault();
+    // Set loading to true when search is initiated
+    setLoading(true); 
+    // Clear any previous errors
+    setError(null); 
+    try {
+        // Fetch artworks by maker from API
+      const results = await apiService.fetchArtworksByMaker(query); 
+      // Set fetched artworks to state
+      setArtworks(results); 
+      // Set loading to false after fetching data
+      setLoading(false); 
+    } catch (error) {
+        // Set error if fetching artworks fails
+      setError(error); 
+      // Set loading to false in case of error
+      setLoading(false); 
+    }
+  };
+
+  return (
+    <Container>
+      <Form onSubmit={handleSearch} className="mb-4">
+        {/* Label for search form */}
+        <Form.Group controlId="searchQuery">
+          <Form.Label>Search Artworks by Title or Artist</Form.Label>
```

Comment:
The heading is a bit greyed out. It is not clearly visible

Reply:
+1